README.md

```diff
@@ -227,90 +259,55 @@ ReactDOM.render(
 - `thisContext: any`: `this` context to use when calling the callback function.
 - => Returns an `off` method to cancel the event subscription.
 
+### `query(options: OperationOptions) => Promise<ExecutionResult>` : Executes GraphQL operation over the WebSocket
+- `options: OperationOptions`:
+    * `query: string` - GraphQL operation as string or parsed GraphQL document node
+    * `variables?: Object` - Object with GraphQL variables
+    * `operationName?: string` - GraphQL operation name
+    * `context?: any` - Execution context for the operation
+    
+### `close() => void` - closes the WebSocket connection manually
```

Comment:
when my PR gets merged this should mention that it closes the websocket manually without firing the tryReconnect, even if its set 😄 #137